modules/ROOT/pages/a11ychecker.adoc

```diff
@@ -393,3 +383,20 @@ console.log(issues);
   }
 ]
 ----
+
+[[issue]]
+=== The `+issue+` object
+
+Each `rule` in a11ychecker will generate an issue on any element which matches it. This issue is available as an object in a number of APIs, and contains the specific data used in the a11ychecker dialog, pertaining to the element which triggered it. With `a11ychecker_filter_issue`, these issues can be parsed before they are classified as issues, with the data provided in the `issue` object used in the callback function. 
+
+The `+issue+` object is provided to any function implementation of this option. This object provides relevant data pertaining to the triggered issue:
```

Comment:
```suggestion
The `+issue+` object is provided to any option that implements said option as a function. This object provides relevant data pertaining to the triggered issue:
```

_Function_ here reads like it should be in the adjectival form, _functional_; describing the form taken by the noun phrase, _implementation of this option_.

I suspect that’s not true, but it’s hard to parse this sentence without thinking the noun is masquerading as a comparative.

The suggestion above is likely wrong but it’s me grasping at re-phrasing this so the original noun phrase and the noun-that-looks-reads-an-adjective can both still be nouns.

The reason I suspect it’s not right is that the re-written noun phrase is now, strictly speaking, a verb phrase with a referent — _implements said option_.

As I said, grasping.

Reply:
```suggestion
The `+issue+` object provides relevant data pertaining to any issue generated by an element which violates an xref:a11ychecker.adoc#accessibility-rules[Accessibility rule]):
```
Ah, I missed this. I started writing this under the `a11ychecker_filter_issue` option, then found it already existed under the `getAudit()` API, so I moved the whole section to the bottom. It's an object which is generated in the audit (when the dialog is opened, or via the API), which is also provided to `a11ychecker_filter_issue` for filtering before the final audit is produced. Let me know if this sounds better.

Reply:
This reads cleanly and succinctly.